Windows/GEDebugger/GEDebugger.cpp

```diff
@@ -22,6 +22,7 @@
 #include <vector>
 
 #include "Common/CommonWindows.h"
+#include <windows.h>
```

Comment:
Please use `#include "Common/CommonWindows.h"` instead which defines WIN32_LEAN_AND_MEAN.  Why did you need to add this twice?

-[Unknown]